### DISCUSSION TITLE
SpaCy with German: Passive, relative clause are not being determined (token.dep_ == sbp, rc)

### DISCUSSION THREAD

**Discussion by dkharlanau** [explosion/spaCy#11094] [Help: Other Questions] [answered]
## How to reproduce the behaviour
```
import spacy

nlp = spacy.load("de_core_news_md"  ) 
for label in nlp.get_pipe("parser").labels:
    print(label, " -- ", spacy.explain(label)) 
```

it shows the available labels for dependency:
............
- sb  --  subject
- sbp  --  passivized subject (PP)**
- svp  --  separable verb prefix
- rc  --  relative clause

when I call the nlp function with the text:
```

doc = nlp("Der Brief wird geschrieben.")

for token in doc:
    print(token.text, 'with dependency:', token.dep_ )
 
doc = nlp("Der Brief ist geschrieben worden")
```
where  _Der Brief_ ist geschrieben worden, Der Brief wird geschrieben._  is in Passiv .

However, it shows that the object's dependency is  sb  --  subject and not sbp  --  passivized subject (PP)
```
Der with dependency: nk
Brief with dependency: **sb**  
wird with dependency: ROOT
geschrieben with dependency: oc
. with dependency: punct

Der  / with dependency: nk
Brief  / with dependency: **sb**
ist  / with dependency: ROOT
geschrieben  / with dependency: oc
worden  / with dependency: oc
```
the same situation for relative clause
```
doc = nlp("Die Kinder, die dort spielen, wohnen in der Schweiz")

for token in doc:
    print(token.text, 'with dependency:', token.dep_ )
```
```
Die with dependency: nk
Kinder with dependency: sb
, with dependency: punct
**die with dependency: sb (??? ) -? rc**
dort with dependency: mo
spielen with dependency: rc
, with dependency: punct
wohnen with dependency: ROOT
in with dependency: mo
der with dependency: nk
Schweiz with dependency: nk
```

Version is 3.3.0
NAME              SPACY                 VERSION
de_core_news_lg   >=3.3.0.dev0,<3.4.0   3.3.0     ✔
de_core_news_md   >=3.3.0.dev0,<3.4.0   3.3.0     ✔
de_core_news_sm   >=3.3.0.dev0,<3.4.0   3.3.0     ✔
de_dep_news_trf   >=3.3.0.dev0,<3.4.0   3.3.0     ✔
en_core_web_sm    >=3.3.0.dev0,<3.4.0   3.3.0     ✔

Can you check what is the problem here? The result is the same independent on the model ( lg,md, trf ).

**Comment by adrianeboyd** [comment, marked as answer]
I think you're misunderstanding how these dependency labels are used.

`rc` is used to attach the head of the relative clause to the noun it's modifying ("Kinder" -> "spielen"):

![Screenshot_2022-07-07_09-08-37](https://user-images.githubusercontent.com/5794899/177713240-1e452843-1c6f-463e-bcc4-fcfdf3c76f74.png)

`sbp` is used for the "von X" phrase (logical subject / agent) in a passive sentence, not the grammatical subject of the passive sentence ("geschrieben" -> "von"):

![Screenshot_2022-07-07_09-06-28](https://user-images.githubusercontent.com/5794899/177713305-d113151a-a786-4785-93d3-826dd294d3e6.png)

The grammatical subject ("Brief") still uses "sb" as in an active sentence.

Here's the full TIGER annotation manual (in German): https://www.ims.uni-stuttgart.de/documents/ressourcen/korpora/tiger-corpus/annotation/tiger_scheme-syntax.pdf

Obviously you'll run into some level of statistical errors in the models (see #3052), and especially for sentences that look less like the training data (newspaper text), but `rc` and `sbp` in these particular examples look correct to me.

**Reply by dkharlanau** [reply]
Thanks a lot for your quick response. With your comment, I agree now. Thank you for the clarification and the link to Tiger annotation.
